patches/third_party-blink-renderer-platform-fonts-font_fallback_list.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
+index 62d4121f7078b90cc08ba72bb53b5c96f73c0a31..417d7e53eb926bee4988e6119059840c83cb01bb 100644
+--- a/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
++++ b/third_party/blink/renderer/platform/fonts/font_fallback_list.cc
+@@ -173,6 +173,7 @@ scoped_refptr<FontData> FontFallbackList::GetFontData(
+           GetFontSelector()->ReportFontLookupByUniqueOrFamilyName(
+               curr_family->FamilyName(), font_description,
+               DynamicTo<SimpleFontData>(result.get()));
++          BRAVE_FONT_FALLBACK_LIST
```

Comment:
this should be BRAVE_GET_FONT_DATA

Reply:
fixed in https://github.com/brave/brave-core/pull/12234/commits/236ac4bfa72c8cb6e79cafdc1d50c34166adbed4